assets/components/contribLegal/contribLegal.jsx

```diff
@@ -0,0 +1,33 @@
+// @flow
```

Comment:
We already have another shared legal/copy component (`termsPrivacy`), in order to have a better folder structure, should we create a new folder inside shared component which can group the two pieces of legal copy? What do you think?